src/wolfsftp.c

```diff
@@ -2870,6 +2870,42 @@ static int wolfSSH_SFTPNAME_readdir(WOLFSSH* ssh, WDIR* dir, WS_SFTPNAME* out,
     return WS_SUCCESS;
 }
 
+#elif defined(WOLFSSH_USER_FILESYSTEM) && defined(SFTP_Name_readdir)
```

Comment:
Would like to see the overridable macro match the naming scheme of all caps with WOLFSSH_ prepended. And near the top of the if-else list not restricted to WOLFSSH_USER_FILESYSTEM.

Reply:
Not sure what you mean with "near the top". Before my patch only `SFTP_GetAttributes` and `SFTP_GetAttributes_Handle` seemed to be overridable, but only if `WOLFSSH_USER_FILESYSTEM` is defined, I tried to stick to that, plus find a way to selectively override the specific function I needed.

Can you clarify?

Reply:
Around here for the top of the if else macro chain, https://github.com/wolfSSL/wolfssh/pull/552/files#diff-03800f8cf7db0a809cc8ba47f57f1e807aa193440295d3822eadbd4255c97118L2528. If someone is building then with say Windows, they can then still override the SFTP_NAME_READDIR

Reply:
@JacobBarthelmeh please, check if it's ok now.